packages/components/src/components/bal-navigation/bal-navigation-menu-list/bal-navigation-menu-list.tsx

```diff
@@ -1,15 +1,46 @@
-import { Component, h, Host, Prop } from '@stencil/core'
+import { Component, h, Host, Listen, Prop, State } from '@stencil/core'
 import { BEM } from '../../../utils/bem'
 import { Props } from '../../../props'
+import { isPlatform } from '../../../utils/platform'
 
 @Component({
   tag: 'bal-navigation-menu-list',
 })
 export class NavigationMenuList {
+  /**
+   * Color of the menu list card background
+   */
   @Prop() color: 'white' | 'grey' = 'white'
+  /**
+   * Optional headline of the menu list card
+   */
   @Prop() headline?: string
+  /**
+   * Optional href of the menu list card headline as link
+   */
   @Prop() href?: string
+  /**
+   * Optional target of the menu list card headline target as link
+   */
   @Prop() target: Props.BalButtonTarget = '_self'
+  @State() headingLevel!: 'h4' | 'h5'
+
+  @Listen('resize', { target: 'window' })
+  async resizeHandler() {
+    this.setHeadingLevel()
+  }
+
+  connectedCallback() {
+    this.setHeadingLevel()
+  }
+
+  setHeadingLevel = () => {
+    if (isPlatform('touch')) {
```

Comment:
Could we do that with SASS to improve the performance?

Reply:
@hirsch88 as discussed, we will leave like it is